TeachingRecordSystem/src/TeachingRecordSystem.Core/Jobs/EWCWalesImportJob.cs

```diff
@@ -0,0 +1,116 @@
+using Azure.Storage.Blobs;
+using TeachingRecordSystem.Core.Dqt;
+using TeachingRecordSystem.Core.Dqt.Queries;
+using TeachingRecordSystem.Core.Jobs.EWCWalesImport;
+
+namespace TeachingRecordSystem.Core.Jobs;
+
+public class EWCWalesImportJob(ICrmQueryDispatcher crmQueryDispatcher, BlobServiceClient blobServiceClient, InductionImporter inductionImporter, QTSImporter qtsImporter)
+{
+    private readonly string _processedFolder = "processed";
+    private readonly string _pickupFolder = "pickup";
+    private readonly string _storageContainer = "legacy-integrations-import";
+
+    public string[] GetImportFiles()
+    {
+        BlobContainerClient containerClient = blobServiceClient.GetBlobContainerClient(_storageContainer);
+        var files = containerClient.GetBlobs().Select(x => x.Name).ToArray();
+        return files;
+    }
+
+    public async Task<MemoryStream> GetDownloadStream(string fileName)
+    {
+        BlobContainerClient containerClient = blobServiceClient.GetBlobContainerClient(_storageContainer);
+        BlobClient blobClient = containerClient.GetBlobClient(fileName);
+        using (var downloadStream = new MemoryStream())
```

Comment:
Don't want a `using` here; `MemoryStream` doesn't do anything in `Dispose()` but if it did we'd be disposing it before it could be used by the consumer.